inc/wpml-compatibility-test-tools-messages.class.php

```diff
@@ -67,15 +67,14 @@ public function __call( $name, $arguments ) {
 				// Prepare a message
 				$message =
 					__( "WPML Compatibility Tester plugin is running and will automatically add language information to all new duplicates for your site. Right now, it will add language information for the following post fields:", 'wpml-compatibility-test-tools'  ) . "<br/>" .
-					( isset( $duplicate_strings['post']['title'] ) 			? '[&#10004;] ' : '[ ] ' ) . __( 'Post title'		  , 'wpml-compatibility-test-tools' ) . "<br/>" .
-					( isset( $duplicate_strings['post']['content'] ) 		? '[&#10004;] ' : '[ ] ' ) . __( 'Post content'		  , 'wpml-compatibility-test-tools' ) . "<br/>" .
-					( isset( $duplicate_strings['post']['image-tags'] )	    ? '[&#10004;] ' : '[ ] ' ) . __( 'Image alt and title', 'wpml-compatibility-test-tools' ) . "<br/>" .
-					( isset( $duplicate_strings['post']['excerpt'] ) 		? '[&#10004;] ' : '[ ] ' ) . __( 'Post excerpt'		  , 'wpml-compatibility-test-tools' ) . "<br/>" .
-					( isset( $duplicate_strings['custom_fields']['value'] ) ? '[&#10004;] ' : '[ ] ' ) . __( 'Custom fields'	  , 'wpml-compatibility-test-tools' ) . "<br/>" .
-					( isset( $duplicate_strings['taxonomy']['all'] )	 	? '[&#10004;] ' : '[ ] ' ) . __( 'Term name '		  , 'wpml-compatibility-test-tools' ) . "<br/>" .
-					( isset( $duplicate_strings['taxonomy_slug']['all'] )	? '[&#10004;] ' : '[ ] ' ) . __( 'Term slug'		  , 'wpml-compatibility-test-tools' ) . "<br/>" .
+					( isset( $duplicate_strings['post']['title'] ) 		   ? '[&#10004;] ' : '[ ] ' ) . __( 'Post title'   , 'wpml-compatibility-test-tools' ) . "<br/>" .
+					( isset( $duplicate_strings['post']['content'] ) 	   ? '[&#10004;] ' : '[ ] ' ) . __( 'Post content' , 'wpml-compatibility-test-tools' ) . "<br/>" .
+					( isset( $duplicate_strings['post']['excerpt'] ) 	   ? '[&#10004;] ' : '[ ] ' ) . __( 'Post excerpt' , 'wpml-compatibility-test-tools' ) . "<br/>" .
+					( isset( $duplicate_strings['custom_field']['value'] ) ? '[&#10004;] ' : '[ ] ' ) . __( 'Custom fields', 'wpml-compatibility-test-tools' ) . "<br/>" .
+					( isset( $duplicate_strings['taxonomy']['all'] )	   ? '[&#10004;] ' : '[ ] ' ) . __( 'Term name '   , 'wpml-compatibility-test-tools' ) . "<br/>" .
+					( isset( $duplicate_strings['taxonomy_slug']['all'] )  ? '[&#10004;] ' : '[ ] ' ) . __( 'Term slug'	   , 'wpml-compatibility-test-tools' ) . "<br/>" .
 					sprintf( "<a href=\"%s\">" . __( "Click here to change fields to duplicate", 'wpml-compatibility-test-tools') . "</a><br/>", admin_url( 'admin.php?page=wctt' ) ) . "<br/>" .
-					__("To proceed, select all the site's content, scroll down and select <strong>Duplicate all</strong> and then click on <strong>Send documents</strong>.", 'wpml-compatibility-test-tools') . "<br/>" .
+					__( "To proceed, select all the site's content, scroll down and select <strong>Duplicate all</strong> and then click on <strong>Send documents</strong>.", 'wpml-compatibility-test-tools' ) . "<br/>" .

```

Comment:
Sorry I am not sure I understand. 
I've only applied WP coding standards to this line (necessary spaces)